crates/bitwarden-exporters/src/cxf/editable_field.rs

```diff
@@ -0,0 +1,225 @@
+use bitwarden_vault::FieldType;
+use credential_exchange_format::{
+    EditableField, EditableFieldBoolean, EditableFieldConcealedString, EditableFieldString,
+    EditableFieldWifiNetworkSecurityType,
+};
+
+use crate::Field;
+
+/// Helper function to create a Field from any EditableField type
+pub(super) fn create_field<T>(name: String, field: &T) -> Field
+where
+    T: EditableFieldToField,
+{
+    Field {
+        name: Some(name),
+        value: Some(field.field_value()),
+        r#type: field.field_type() as u8,
+        linked_id: None,
+    }
+}
+
+/// Trait to convert EditableField types to Field values and types
+pub(super) trait EditableFieldToField {
+    fn field_value(&self) -> String;
+    fn field_type(&self) -> FieldType;
+}
+
+impl EditableFieldToField for EditableField<EditableFieldString> {
+    fn field_value(&self) -> String {
+        self.value.0.clone()
+    }
+
+    fn field_type(&self) -> FieldType {
+        FieldType::Text
+    }
+}
+
+impl EditableFieldToField for EditableField<EditableFieldConcealedString> {
+    fn field_value(&self) -> String {
+        self.value.0.clone()
+    }
+
+    fn field_type(&self) -> FieldType {
+        FieldType::Hidden
+    }
+}
+
+impl EditableFieldToField for EditableField<EditableFieldBoolean> {
+    fn field_value(&self) -> String {
+        if self.value.0 { "true" } else { "false" }.to_string()
```

Comment:
booleans already get stringified as `true/false`, so we can just call tostring on it directly
```suggestion
        self.value.0.to_string()
```